editors/code/tsconfig.json

```diff
@@ -6,6 +6,8 @@
         "lib": [
             "es2019"
         ],
+        "esModuleInterop": true,
```

Comment:
This is required, otherwise we have some problems with bundling with `node-fetch`. Didn't dive into all that nightmare, just turned on these two magical toggles that I always have in any TypeScript project anyway ...